mindee/product/invoice_splitter/invoice_splitter_v1.py

```diff
@@ -21,9 +21,7 @@ def __init__(self, raw_prediction: StringDict) -> None:
         self.prediction = InvoiceSplitterV1Document(raw_prediction["prediction"])
         self.pages = []
         for page in raw_prediction["pages"]:
-            self.pages.append(
-                Page(InvoiceSplitterV1Document, page, page["id"], page["orientation"])
-            )
+            self.pages.append(Page(InvoiceSplitterV1Document, page))
```

Comment:
so much nicer :+1:

Reply:
Yeah, not a 100% sure why it was that way in the first place? :thinking: